frontend/packages/kubevirt-plugin/integration-tests/tests/models/kubevirtDetailView.ts

```diff
@@ -11,30 +12,32 @@ import { DetailView } from './detailView';
 export class KubevirtDetailView extends DetailView {
   async getAttachedDisks(): Promise<StorageResource[]> {
     await this.navigateToTab(TABS.DISKS);
-    const resources = [];
-    for (const row of await resourceRows) {
-      const cells = row.$$('div');
+    const resources: StorageResource[] = [];
+    const rows = await kubevirtDetailView.tableRows();
+    rows.forEach((line) => {
+      const cols = line.split(/\s+/);
```

Comment:
can you please explain how is the table text and rows are resolved? How come we get the columns  just by splitting it by spaces?

Reply:
Well calling `getText()` on the table returns string in the following format
```
attr00 attr01 ... attr0n\n
attr10 attr11 ... attr1n\n
```
So `forEach` iterates over lines and thus I can split the columns by spaces.